app/assets/javascripts/partials/template_edit.html.erb

```diff
@@ -13,7 +13,7 @@
 
 <div class="row no-print" style="margin-top: 2em;" ng-show="loaded">
   <div class="col-md-10 col-md-offset-1">
-    <input id="checklist-category" placeholder="Add a Category" ng-model="template.template_category.name" type="text" class="form-control" typeahead-editable="false" typeahead="category.name for category in categories | filter:$viewValue | limitTo: 5" typeahead-on-select="saveCategory($item)" ng-blur="newCategory($event)"></input>
+    <input id="checklist-category" placeholder="Add a Tag" ng-model="template.template_category.name" type="text" class="form-control" typeahead-editable="false" typeahead="category.name for category in categories | filter:$viewValue | limitTo: 5" typeahead-on-select="saveCategory($item)" ng-blur="newCategory($event)"></input>
```

Comment:
I see. Let's talk about this.

I was more referring to needing a line of explanation near the input tag to explain how categories work because, even as one of the folks who wrote it, I didn't understand how it worked.

Do change it back to category in the meantime though.